docker/images/localdev-server/entrypoint.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+sudo chown "$LOCALDEV_USER" /var/run/docker.sock
```

Comment:
We could add `sudo chown -R "$LOCALDEV_USER" /var/run/user` as well.

Reply:
pushed this fix.